generator/src/Data/String/Multiline.purs

```diff
@@ -0,0 +1,7 @@
+module Data.String.Multiline where
+
+import Data.String (joinWith)
+
+multiline :: Array String -> String
```

Comment:
Maybe it will be better if both `capitalize` and `multiline` will live in one module?